x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/audit/logfile/LoggingAuditTrailFilterTests.java

```diff
@@ -1734,12 +1684,9 @@ public void testRolesFilter() throws Exception {
         final String[] unfilteredRoles = _unfilteredRoles.toArray(new String[0]);
         Authentication authentication;
         if (randomBoolean()) {
-            authentication = createAuthentication(
-                new User("user1", new String[] { "r1" }, new User("authUsername", new String[] { "r2" })),
-                "effectiveRealmName"
-            );
+            authentication = createAuthentication(new User("user1", "r1"), new User("authUsername", "r2"), "effectiveRealmName");
```

Comment:
This is an example where a single User (with inner user) argument is split into 2 User arguments.